src/server/req_manager.c

```diff
@@ -1649,13 +1651,16 @@ mgr_server_unset(struct batch_request *preq)
 				ATTR_schediteration) == 0) {
 			if (dflt_scheduler) {
 				svrattrl *tm_list;
-				tm_list = attrlist_create(plist->al_name, NULL, 0);
+				/* value is 600 so it is of size 4 including terminating character */
+				tm_list = attrlist_create(plist->al_name, NULL, 4);
 				if (tm_list == NULL) {
 					reply_badattr(-1, bad_attr, plist, preq);
 				}
 				tm_list->al_link.ll_next->ll_struct = NULL;
-				rc = mgr_unset_attr(dflt_scheduler->sch_attr, sched_attr_def, SCHED_ATR_LAST, tm_list,
-					-1, &bad_attr, (void *)dflt_scheduler, PARENT_TYPE_SCHED, INDIRECT_RES_CHECK);
+				/* when unset, set scheduler_iteration to 600 seconds */
+				sprintf(tm_list->al_value, "%d", 600);
+				rc = mgr_set_attr(dflt_scheduler->sch_attr, sched_attr_def, SCHED_ATR_LAST, tm_list,
```

Comment:
I'm don't know much about this code, but we will do an unset operation after this while loop is over, so does doing a set here make any difference?